apps/dashboard/src/components/activity/activity-job-item.tsx

```diff
@@ -105,7 +111,7 @@ function getStatusMessage(job: IActivityJob): string | React.ReactNode {
   }
 
   if (job.status === JobStatusEnum.CANCELED) {
-    return 'Step was skipped by step conditions';
+    return 'Step was skipped';
```

Comment:
I wonder, is it possible to show here more contextual snippet for the reason? For example step skipped due to lacking credentials, etc... would be just faster for the user to understand what happens.

Reply:
@scopsy currently we don't save any reason on the job, but should be possible to do

Reply:
tend to agree with Dima, this one is much more informative, altho it is wrong because it was filtered by preferences. :( 
<img width="476" height="280" alt="image" src="https://github.com/user-attachments/assets/a39178f3-67a2-4f47-8420-d5a126ea0a3b" />
maybe we can pass more data to the raw data and display it as well?